runtime/src/comm/gasnet/comm-gasnet-ex.c

```diff
@@ -723,8 +725,10 @@ int32_t chpl_comm_getMaxThreads(void) {
 //
 static volatile int pollingRunning;
 static volatile int pollingQuit;
-static chpl_bool pollingRequired;
+static chpl_bool pollingRequired = false;
 static atomic_spinlock_t pollingLock;
+static chpl_bool usePSHM = false;
+static chpl_bool pshmInUse = false;
```

Comment:
@jhh67 I strongly object to this change that you just folded into a rebase, which (given the current value of `usePSHM = false`) has the effect of disabling PSHM by default. I think this effectively defeats one of the primary goals of this PR, which is to provide fast co-locale comms by default.

Was this change unintentional?

```suggestion
static chpl_bool usePSHM = true;
static chpl_bool pshmInUse = false;
```